Model/Transaction.php

```diff
@@ -373,6 +373,12 @@ protected function getParentAmounts($attr, $items, $type = 'order') {
      */
     protected function getProductTaxCode($item, $order)
     {
+        // Check for a PTC saved to the Item
+        if ($item->getTjPtc()) {
```

Comment:
@ryanreeves-taxjar If a product is set to the "Fully Taxable" category and we save an empty string in the `tj_ptc` column, will this conditional be truthy and return back an empty tax code? Or will it fall back to looking up the tax code on the product? Checking to ensure an item that goes from "Fully Taxable" to another category before transaction sync remains fully taxable.

Reply:
@jakedjohnson NULL values are stored when no PTC is present, which does cause `if ($item->getTjPtc())` to equate to false.  This causes a potentially incorrect second lookup (the behavior I mentioned in chat).  

One solution is to save a non-empty value that represents no PTC.  I don't think we have a "Fully Taxable" PTC, so maybe 00000 as the opposite of 99999?  Then:

- If the column is empty, the order_item was created before the `tj_ptc` field existed and fall back to the product
- If it's not empty, only use the 'tj_ptc' field

Thoughts?

Reply:
@ryanreeves-taxjar Feel free to use `11111` for the "Fully Taxable" category if that helps prevent any issues here. It's equivalent to not passing a tax code or a blank string.